src/git.ts

```diff
@@ -736,11 +780,13 @@ export function handleDiffFiles(actionData: ActionData): string | undefined {
     const ref1: string | undefined = actionData.params.ref1;
     const ref2: string | undefined = actionData.params.ref2;
     const filePath: string = actionData.params.filePath ?? '.';
+    if (!isPathNeuroSafe(filePath)) {
+        NEURO.client?.sendContext('The provided file path is not allowed.');
+        return;
+    }
+    const diffThisFile = getAbsoluteFilePath(filePath);
```

Comment:
`isPathNeuroSafe` only works with absolute paths (this isn't documented because I forgot)
```suggestion
    const diffThisFile = getAbsoluteFilePath(filePath);
    if (!isPathNeuroSafe(diffThisFile)) {
        NEURO.client?.sendContext('The provided file path is not allowed.');
        return;
    }
```